obitos_convert.py

```diff
@@ -27,20 +28,40 @@ def convert_file(filename):
         )[1]
         row["epidemiological_week_2020"] = brazilian_epidemiological_week(date)[1]
 
+        # There are some missing days on the registral, so default to zero
```

Comment:
If the number is not available, the value should be `None` instead.

Reply:
This is ok so? empty fields?

```
2020-01-01,AC,1,1,0,3,2,1,0,4,0,2,1,1,1,7,,,3,2,1,,4,,2,1,1,1,7,,10,12,10,12
2020-01-02,AC,1,1,0,0,0,1,0,6,0,2,0,2,0,6,,,,,2,,10,,4,,3,,13,,7,10,17,22
2020-01-03,AC,1,1,0,0,0,2,0,6,0,3,0,2,0,8,,,,,4,,16,,7,,5,,21,,8,13,25,35
2020-01-04,AC,1,1,0,3,0,1,0,1,0,0,1,1,0,4,,,6,,5,,17,,,2,6,,25,,5,6,30,41
2020-01-05,AC,1,2,0,1,0,0,0,5,0,4,1,0,0,4,,,7,,,,22,,11,3,,,29,,6,9,36,50
```